src/App.js

```diff
@@ -1,12 +1,43 @@
-import { React, Component } from 'react';
-
-
+import React, { Component } from 'react';
+import { Route, Switch, Link } from 'react-router-dom';
+import Header from './components/Header';
+import LoginPage from './pages/LoginPage';
+import HomePage from './pages/HomePage';
+import EmployeesPage from './pages/EmployeesPage';
+import NotFoundPage from './pages/NotFoundPage';
+import PrivateRoute from './components/PrivateRoute';
```

Comment:
PrivateRoute should not be in components folder, create another folder for different route types